### PR TITLE
Create tags decorator to tests

### PR DESCRIPTION
**Pull request by cmagnobarbosa**
Today, all the tests are executed without distinction. This commit creates a tag to divide tests in type (unit, integration, e2e) and in size (small, medium, large). Small and unit tests are the standard parameters.

To set tags to a test function:

```python
from tests.helpers import tags

@tags(type='unit', size='small')
def test_function():
...
```

To run the tests:
```shell
  python setup.py ci --size=medium --type=unit
```

The _'test'_ parameter uses the standard arguments.

Related https://github.com/kytos/kytos/issues/1041

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -35,6 +35,8 @@
 
 # NApps enabled by default
 # CORE_NAPPS = ['of_core']
+os.environ["KYTOS_TESTS_SIZE"] = 'small'
+os.environ["KYTOS_TESTS_TYPE"] = 'unit'
```
Comment (hdiogenes) [inline setup.py]: Shouldn't it be `os.environ.get("KYTOS_TESTS_SIZE", 'small')`?

Comment (cmagnobarbosa) [reply setup.py]: Solved moving the default value to decorator.

File: setup.py [hunk 2]
```diff
@@ -55,6 +55,28 @@ def finalize_options(self):
...
+
+    def initialize_options(self):
+        """Set default size and type decorator tests."""
+        self.size = 'small'
+        self.type = 'unit'
+
+    def finalize_options(self):
+        """Post-process."""
+        assert self.size in ('small', 'medium', 'large'), 'Invalid size.'
+        assert self.type in ('unit', 'integration', 'e2e'), 'Invalid type.'
```
Comment (hdiogenes) [inline setup.py:75]: Suggestion: f'Invalid test type: {self.type}'

File: setup.py [hunk 3]
```diff
@@ -86,21 +108,21 @@ class Linter(SimpleCommand):
     def run(self):
         """Run yala."""
         print('Yala is running. It may take several seconds...')
-        check_call('yala *.py v0x??/*.py tests/*.py tests/unit/*.py'
-                   ' tests/v0x??/*.py',
+        check_call('yala *.py',
```
Comment (hdiogenes) [inline setup.py:111]: Is this right? Will it really test all files?

**Comment by cmagnobarbosa** [comment]
Moved to  #69

**Pull request closed without merging**